examples/gallery/symbols/multi_parameter_symbols.py

```diff
@@ -3,48 +3,87 @@
 -------------------------
 
 The :meth:`pygmt.Figure.plot` method can plot individual multi-parameter
-symbols by passing the corresponding shortcuts listed below to the ``style``
-parameter. Additionally, we must define the required parameters in a 2d list or
-numpy array (``[[parameters]]`` for a single symbol or
-``[[parameters_1],[parameters_2],[parameters_i]]`` for several ones) or use an
-appropriately formatted input file and pass it to ``data``.
-
-The following symbols are available:
-
-- **e**: ellipse, ``[[lon, lat, direction, major_axis, minor_axis]]``
-- **j**: rotated rectangle, ``[[lon, lat, direction, width, height]]``
-- **r**: rectangle, ``[[lon, lat, width, height]]``
-- **R**: rounded rectangle, ``[[lon, lat, width, height, radius]]``
-- **w**: pie wedge, ``[[lon, lat, radius, startdir, stopdir]]``, the last two
-  arguments are directions given in degrees counter-clockwise from horizontal
-
-Upper-case versions **E**, **J**, and **W** are similar to **e**, **j** and
-**w** but expect geographic azimuths and distances.
+symbols by passing the corresponding shortcuts (**e**, **j**, **r**, **R**,
+**w**) to the ``style`` parameter:
+
+- **e**: ellipse
+- **j**: rotated rectangle
+- **r**: rectangle
+- **R**: rounded rectangle
+- **w**: pie wedge
+
 """
 
+# sphinx_gallery_thumbnail_number = 2
+
 import pygmt
 
+########################################################################################
+# We can plot multi-parameter symbols using the same symbol style. We need to
+# define locations (lon, lat) via the ``x`` and ``y`` parameters (scalar for
+# a single symbol or 1d list for servel ones) and two or three symbol
+# parameters after those shortcuts via the ``style`` parameter.
+#
+# The symbol parameters in the ``style`` parameter are defined as:
```

Comment:
Minor suggestion since there are other, simpler arguments for the ``style`` parameter.
```suggestion
# The multi-parameter symbols in the ``style`` parameter are defined as:
```